src/main/drivers/accgyro/accgyro_spi_mpu9250.c

```diff
@@ -75,6 +75,7 @@ static bool mpu9250SpiSlowReadRegisterBuffer(const busDevice_t *bus, uint8_t reg
     return true;
 }
 
+#if 0 // XXX Not used at all. Debugging aid?
```

Comment:
Do we really need nested `#if 0` here? Should this just be removed, considering the comment says it doesn't work?

Reply:
I think removal of the `mpu9250SpiResetGyro` and related (including `mpuConfiguration_s.resetFn`) better go into another PR, after finding out what was actually intended with the `resetFn`.

Reply:
Then please convert this into a named `#if defined()`, so that it can at least be built with a command line option. Keeping this in as decoration serves no purpose.